### PR TITLE
feat: add session times and conference start time to features

### PR DESCRIPTION
**Pull request by gabiborlea**

### CODE REVIEW

File: src/database/FirehoseConnector.js [hunk 1]
```diff
@@ -206,6 +209,9 @@ class FirehoseConnector {
             meetingUrl,
             meetingUniqueId,
             endpointId,
+            conferenceStartTime,
```
Comment (andrei-gavrilescu) [inline src/database/FirehoseConnector.js:212]: I think these need to be formatted to sql time with getSQLTimestamp

File: src/database/FirehoseConnector.js [hunk 2]
```diff
@@ -206,6 +209,9 @@ class FirehoseConnector {
...
+            sessionStartTime,
```
Comment (andrei-gavrilescu) [inline src/database/FirehoseConnector.js:213]: Update the redshift-schema.sql file with the new fields

File: src/database/FirehoseConnector.js [hunk 3]
```diff
@@ -206,6 +209,9 @@ class FirehoseConnector {
...
+            sessionEndTime,
             sessionDurationMs,
             conferenceDurationMs,
```
Comment (nils-ohlmeier) [inline src/database/FirehoseConnector.js:216]: If we expose now the start and end times, do we still need to calculate and expose the duration's?
Or could we just drop the code for the duration's as these will get calculated on the SQL anyway?

Comment (andrei-gavrilescu) [reply src/database/FirehoseConnector.js:216]: Fair point, but one thing to consider is that the data team uses conferenceDuration in a bunch of their queries, and having to calculate it for each entry might be costly when doing the queries so maybe we keep it in order to have more efficient queries in redshift in the detriment of having the logic in rtcstats?

File: src/database/FirehoseConnector.js [hunk 4]
```diff
@@ -189,6 +192,9 @@ class FirehoseConnector {
         } = features;
 
         const createDate = getSQLTimestamp();
+        const conferenceStartTime = getSQLTimestamp(conferenceStartTimestamp);
```
[the remark this answers is not in the document]
Comment (gabiborlea) [reply src/database/FirehoseConnector.js:195]: yeah, right

[the remark this answers is not in the document]
Comment (andrei-gavrilescu) [reply src/database/FirehoseConnector.js]: I think the same applies for sessionStatsTime/endtTime as well

**Review by andrei-gavrilescu — APPROVED** [review]

**Pull request merged**